src/qtgui/plotter.cpp

```diff
@@ -1579,7 +1581,8 @@ int CPlotter::xFromFreq(qint64 freq)
 qint64 CPlotter::freqFromX(int x)
 {
     double ratio = (double)x / (double)width();
-    qint64 f = (m_CenterFreq + m_FftCenter - m_Span / 2) + ratio * m_Span;
+    qint64 f = llround((double)m_CenterFreq + (double)m_FftCenter
+                       - (double)m_Span / 2 + ratio * (double)m_Span);
```

Comment:
mey-be, just for consistenci, use `2.0` here as well (used in the above).

Reply:
It really makes no difference since `m_Span` is already cast to double. I could make them all the same to make it prettier.

Reply:
thats it, it is just NIT to make ti prettier, LGTM from me.